src/components/thumbnail-card/__tests__/ThumbnailCardDetails.test.js

```diff
@@ -1,13 +1,12 @@
 // @flow
 import * as React from 'react';
 
-import { mount, shallow } from 'enzyme';
+import { fireEvent, render, screen } from '@testing-library/react';
 
 import * as libDom from '../../../utils/dom';
-
 import ThumbnailCardDetails from '../ThumbnailCardDetails';
 
-const getWrapper = (props = {}) => shallow(<ThumbnailCardDetails title={<div>Foo Bar!</div>} {...props} />);
+const getWrapper = (props = {}) => render(<ThumbnailCardDetails title={<div>Foo Bar!</div>} {...props} />);
```

Comment:
```suggestion
const renderComponent = (props = {}) => render(<ThumbnailCardDetails title={<div>Foo Bar!</div>} {...props} />);
```
to match the other RTL tests